cypress/e2e/tests/OrangeHRM/Leave/leaveRequest.spec.cy.ts

```diff
@@ -0,0 +1,47 @@
+import addNewEmployee from "../../../../support/helpers/OrangeHRM/EmployeeHelper";
+import LoginPage from "../../../../support/pageObjects/OrangeHRM/LoginPage";
+import addEmployee from "../../../../support/pageObjects/OrangeHRM/addEmployee";
+import employeePersonalInfo from "../../../../support/pageObjects/OrangeHRM/employeePersonalInfo";
+import employeeTable from "../../../../support/pageObjects/OrangeHRM/employeeTable";
+const LoginObj: LoginPage = new LoginPage();
+const addEmployeeObj: addEmployee = new addEmployee();
+const employeePersonalInfoObj: employeePersonalInfo = new employeePersonalInfo();
+const employeeTableObj: employeeTable = new employeeTable();
+
+describe("Leave: Leave Requests Process", () => {
+    beforeEach(() => {
+        cy.intercept("/web/index.php/dashboard/index").as("LoginPage");
+        cy.visit("/");
+        LoginObj.login("Admin", "admin123");
+        addEmployeeObj.selectPIM();
```

Comment:
Why you call this action here ? The Employee Added By API , so no need to select PIM